.circleci/config.yml

```diff
@@ -108,6 +126,8 @@ pr_build: &pr_build
             api_docker_build,
             pdf_docker_build,
             seeding_docker_build,
+            perfplat-lambda-docker-worker-build,
+            #perfplat-lambda-docker-api-build
```

Comment:
Can this be uncommented?

Reply:
agree - this should be a dependancy of this one

Reply:
I didn't want the build failing while these weren't in production. But will remove the comments on both.